source/adapters/native_cpu/program.cpp

```diff
@@ -87,6 +87,26 @@ urProgramLink(ur_context_handle_t hContext, uint32_t count,
   DIE_NO_IMPLEMENTATION
 }
 
+UR_APIEXPORT ur_result_t UR_APICALL urProgramCompileExp(ur_program_handle_t,
```

Comment:
These new entry points should be added to the DDI tables in `ur_interface_loader.cpp`